p2panda-store/src/memory_store.rs

```diff
@@ -17,18 +19,25 @@ type RawHeader = Vec<u8>;
 type LogMeta = (SeqNum, Timestamp, Hash);
 type StoredOperation<L, E> = (L, Header<E>, Option<Body>, RawHeader);
 
+/// An in-memory store for core p2panda data types: `Operation` and `Log`.
 #[derive(Clone, Debug)]
 pub struct InnerMemoryStore<L, E> {
```

Comment:
It feels a bit strange to have this as part of our public API, but maybe we want to offer that level of access?

The alternative would be to make `InnerMemoryStore`, `read_store()` and `write_store()` private and only allowing interaction via the API of `MemoryStore` itself.

Reply:
I'm undecided at this point. It feels not very useful at this point to expose it, on the other hand I can see how people would like to re-use the store implementation and add custom methods on it. And wouldn't it be possible to also add additional methods for example for extensions?